### PR TITLE
refactor(frontend): rename types of Ethereum transactions store

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

Renaming Ethereum transactions store types.

### CODE REVIEW

File: src/frontend/src/eth/stores/eth-transactions.store.ts [hunk 1]
```diff
@@ -3,19 +3,19 @@ import type { Transaction } from '$lib/types/transaction';
...
+export interface EthTransactionsStore extends Readable<EthTransactionsData> {
 	set: (params: { tokenId: TokenId; transactions: Transaction[] }) => void;
 	add: (params: { tokenId: TokenId; transactions: Transaction[] }) => void;
 	update: (params: { tokenId: TokenId; transaction: Transaction }) => void;
 	reset: () => void;
 }
 
-const initTransactionsStore = (): TransactionsStore => {
-	const INITIAL: TransactionsData = {} as Record<TokenId, Transaction[]>;
+const initTransactionsStore = (): EthTransactionsStore => {
```
Comment (peterpeterparker) [inline src/frontend/src/eth/stores/eth-transactions.store.ts:15]: You gonna rename `initTransactionsStore` in another PR?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/eth/stores/eth-transactions.store.ts (hunk 1 above)
Comment (AntonioVentilii) [reply src/frontend/src/eth/stores/eth-transactions.store.ts]: sorry, forgot, done in this one

**Comment by AntonioVentilii** [comment]
[OISY-130]

**Pull request merged**